src/platform/plugins/shared/controls/public/controls/data_controls/options_list_control/components/options_list_popover_action_bar.tsx

```diff
@@ -84,7 +84,7 @@ export const OptionsListPopoverActionBar = ({
     selectedOptions = [],
     totalCardinality,
     field,
-    fieldName,
+    displayName,
```

Comment:
The `fieldName` subject is only used in the UI component to generate aria labels and such. Because ES|QL controls won't have a `fieldName` this is being changed to `displayName` and derived within the factory from whichever descriptive properties are available.

The default is now `title` if defined, which I think was an oversight in the initial implementation.